distinguished_contributors.md

```diff
@@ -0,0 +1,16 @@
+# Jupyter Distinguished Contributors
+## Purpose
+This document describes the Distinguished Contributors body of the Jupyter Governance Model. Membership in the Distinguished Contributors is meant to recognize the work of community members that have gone above-and-beyond in their work on the project. It is not meant as a means of conveying power or responsibility. A public record of each Distinguished Contributor’s current biography and achievements will be maintained by the members themselves and will be displayed on the Jupyter website.
+## Rights of Distinguished Contributors
+Distinguished Contributors have the right to elect new Distinguished Contributors. They also have the right to participate in other occasional votes specified by the broader Jupyter Governance Model.
+##Distinguished Contributor membership
+### Criteria for Nominating New Distinguished Contributors
+To be considered for nomination as a Distinguished Contributor a Project Contributor must have produced contributions that are substantial in quality and quantity, and sustained over at least two years. When considering potential Members, the existing Distinguished Contributors will look at nominees with a comprehensive view of their contributions. This will include but is not limited to code, code review, infrastructure work, mailing list and chat participation, community help/building, education and outreach, fundraising, branding, marketing, inclusion and diversity, UX design and research, etc. We are deliberately not setting arbitrary quantitative metrics (like “100 commits in this repo”) to avoid encouraging behavior that plays to the metrics rather than the project’s overall well-being. 
+### Term length
+Membership in the Distinguished Contributors body is a lifetime appointment. 
+### Selecting Distinguished Contributors Members
+Distinguished Contributors are added to this body on an annual basis, through a voting process carried out by the pre-existing body of Distinguished Contributors. The number of candidates chosen each year will be limited to five new members. Nominations for election can be submitted by any Distinguished Contributor to be reviewed by a committee of peer reviewers chosen from the Distinguished Contributors. This committee is responsible for managing the private voting process and reviewing nominees to ensure that they meet the minimum criteria for Distinguished Contributor status. After nominees are finalized, the Distinguished Contributors will vote in a ranked preference process. 
+
+Each newly inducted member will be awarded a nominal purse of up to $500 per awardee, subject to budgetary constraints, in recognition for service to the community.
```

Comment:
What exactly does "nominal purse" mean? Presumably we shouldn't fix a single amount (if we do lower, then it looks bad, and over time presumably inflation makes $500 seem less and less anyway?). To me, the actual dollar amount seems to be an implementation detail, and dependent on funding.

```suggestion
Each newly inducted member will be awarded a nominal purse per awardee, subject to budgetary constraints, in recognition for service to the community.
```

Reply:
This is a point that came up in conversation and we went back and forth on it. I think I agree with your interpretation, Jason. If I recall correctly, the counterpoint was that it shouldn't need to be a conversation every single year. But I may be misremembering other nuances. But perhaps someone else remembers other reasons we put this in here.

Reply:
Fair enough, but it does say "up to", so it already is a conversation each year. My guess is that some nominal amount will be set initially, and then tradition will carry it on for quite a while. But tradition is easier to update than encoded tradition (governance docs).

Reply:
In fact, you could take out the whole sentence of the purse, and leave even the idea of a prize to tradition.